readme.md

```diff
@@ -142,19 +140,17 @@ See the [ESLint docs](http://eslint.org/docs/user-guide/configuring#extending-co
 
 **Note**: This config will also enable the correct [parser options](http://eslint.org/docs/user-guide/configuring#specifying-parser-options) and [environment](http://eslint.org/docs/user-guide/configuring#specifying-environments).
 
-
 ## Maintainers
 
 - [Sindre Sorhus](https://github.com/sindresorhus)
 - [Sam Verschueren](https://github.com/SamVerschueren)
 - [futpib](https://github.com/futpib)
 - [Adam Babcock](https://github.com/MrHen)
 
-###### Former
+### Former
```

Comment:
In this case, these were intended, manual changes because markdownlint threw me a couple of warnings e.g. https://github.com/DavidAnson/markdownlint/blob/v0.16.0/doc/Rules.md#md001

But I am fine with reverting back to how it was before, however ;)

Reply:
I'm okay making the changes in a separate PR.